build/webpack.config.js

```diff
@@ -53,7 +53,15 @@ module.exports = {
       resolve('src'),
       resolve('node_modules'),
     ],
-    alias: config.aliases,
+    alias: {
+      quasar: 'quasar-framework',
+      'quasar-vue-plugin': 'quasar-framework/src/vue-plugin',
+      '@': resolve(__dirname, '../src'),
+      '>': resolve(__dirname, '../test'),
+      variables: resolve(__dirname, '../src/themes/quasar.variables.styl'),
+      slidetoggle: resolve(__dirname, '../src/themes/karrot.slidetoggle.styl'),
+      editbox: resolve(__dirname, '../src/themes/karrot.editbox.styl'),
```

Comment:
I know these were here before, but doesn't seem quite the right things to be aliases...